### DISCUSSION TITLE
Foreign language characters in commit messages

### DISCUSSION THREAD

**Discussion by RalphGauer-Paul** [intersystems/git-source-control#621] [Q&A] [answered]
Hi!
What do I have to change to let the WebUI accept entered foreign language characters like umlauts into the "Enter commit message (required)" field so that the commit message is displayed correctly later?
Older versions of git-source-control (without version number displayed in the WebUI) showed a yellow warning that UTF-8 wasn't configured for Git, but the message was saved correctly.
Current versions of git-source-control don't show a warning but a green success message, containing the umlaut "ä" as the two raw UTF-8 source characters "Ã¤". This wrongly encoded message is also saved into git.
Umlauts entered as commit message into the VSCode commit dialog using git-source-control are saved correctly.
What did I do wrong?

**Comment by isc-pbarton** [comment, marked as answer]
Hello! This appears to be a regression introduced in version 2.4.1. I have logged an issue here to fix it: https://github.com/intersystems/git-source-control/issues/627
